src/message.js

```diff
@@ -424,14 +424,14 @@ export class Message {
    * Sign the message (the literal data packet of the message)
    * @param  {Array<module:key.Key>} privateKeys      private keys with decrypted secret key data for signing
    * @param  {Signature} signature                    (optional) any existing detached signature to add to the message
+   * @param  {Array<module:type/keyid>} signingKeyIds (optional) array of key IDs to use for signing. Each signingKeyIds[i] corresponds to privateKeys[i]
    * @param  {Date} date                              (optional) override the creation time of the signature
    * @param  {Array} userIds                          (optional) user IDs to sign with, e.g. [{ name:'Steve Sender', email:'steve@openpgp.org' }]
    * @param  {Boolean} streaming                      (optional) whether to process data as a stream
-   * @param  {Array<module:type/keyid>} signingKeyIds (optional) array of key IDs to use for signing. Each signingKeyIds[i] corresponds to privateKeys[i]
    * @returns {Promise<Message>}                      new message with signed content
    * @async
    */
-  async sign(privateKeys = [], signature = null, date = new Date(), userIds = [], streaming = false, signingKeyIds = []) {
+  async sign(privateKeys = [], signature = null, signingKeyIds = [], date = new Date(), userIds = [], streaming = false) {
```

Comment:
`CleartextMessage.sign` and `signDetached` need to be changed as well I think @twiss? We should add some more tests probably, since the issue was not detected.

Reply:
Yes, you're right. @alrichry Sorry for the endless stream of comments, but would you mind adding this parameter there, as well? These methods are called when passing a `CleartextMessage.fromText('test')` to `openpgp.sign`, for example. After that I think it should be ready to merge :blush:

Reply:
You got it @twiss 😎, no worries, it's all for quality assurance :)

Reply:
Thanks a lot!